tsconfig.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
This file has been added in PR for DFPL-2289 which is ready to merge, may be delete this file to be added? You can add any properties if needed once that PR is merged to avoid any merge conflicts